jonprl-mode.el

```diff
@@ -48,6 +48,22 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
```

Comment:
This notation seems to me to be a bit more obscure than the others - is it something that there's a wide consensus about? (sorry to be a stick in the mud, but I'd like to make sure that the crazy unicode isn't totally overdone :-) )